cmd/cli/flavor.go

```diff
@@ -18,25 +18,25 @@ import (
 
 func flavorPluginCommand(plugins func() discovery.Plugins) *cobra.Command {
 
-	name := ""
 	var flavorPlugin flavor.Plugin
 
 	cmd := &cobra.Command{
 		Use:   "flavor",
 		Short: "Access flavor plugin",
-		PersistentPreRunE: func(c *cobra.Command, args []string) error {
+	}
+	name := cmd.PersistentFlags().String("name", "", "Name of plugin")
```

Comment:
It does need to be, as you want to specify a plugin `name` for most of the subcommands being used.  IMHO it really should be a positional (required) argument rather than a flag, but that's not something i'm willing to tackle in this patch.